src/components/blocks/card/_card.scss

```diff
@@ -211,11 +213,11 @@
     }
   }
 
-  &:visited &__info-container {
+  &:visited {
```

Comment:
can this rule be moved up to line 20 please @nickbergquist, together with the focus and more associated with its base class, rather than being lost after all the child classes. it's a pattern used everywhere in our code, and much improves readability.